compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -177,9 +178,11 @@ std::set<std::string> gather_customcode_set(const ::tflchef::ModelRecipe &model_
   std::set<std::string> customcode_set;
   for (const auto &operation : model_recipe.operation())
   {
-    auto op_chef = op_chef_registry().lookup(operation.type()).create(&operation);
-    if (op_chef->code() == tflite::BuiltinOperator_CUSTOM)
-      customcode_set.insert(operation.type());
+    if ((operation.has_extype() && operation.extype() == "Custom") || operation.type() == "Custom")
+    {
+      assert(operation.has_custom_code());
```

Comment:
On second thought, this assert always will be true because `custom_code` is optional element, which means that it has a default value. So, this and the assert in the `check_custom_op_value` method can be removed.

Reply:
The check should change into check if given string is empty - default value.

Reply:
@mhs4670go 

Thank you for your comment ! :)
I applied it regarding comments, 
Could you look at it again ?